content/blog/2021/06/kwctl-intro-for-kubernetes-administrators.md

```diff
@@ -0,0 +1,277 @@
+and are going to be removed from Kubernetes 1.25.
+
+Alice wants to find an alternative to PSP. Today she will work to replace
+the [container capabilities PSP](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container)
+with a Kubewarden policy.
+
+We will tag along with Alice, watch all the steps she has to perform
+and learn how `kwctl` will help her.
+
+## Finding the right policy
+
+As a first step, Alice visits [Kubewarden’s Policy Hub](https://hub.kubewarden.io),
+the place where Kubewarden policies can be made discoverable.
+
+Alice enters the *"capabilities"* query and finds a policy that seems to be
+doing exactly what she's looking for. The name of the policy is
+*"psp-capabilities"*.
+
+## Obtaining the policy
+
+The first step for Alice is to obtain the policy. This can be done with a
+simple command:
+
+```shell
+$ kwctl pull registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4
+```
+
+This downloads the policy and stores it under Alice's home directory.
+
+## Understanding how the policy works
+
+Alice now needs to understand how to use the policy. Each policy listed on
+Kubewarden's Policy Hub has links pointing to their documentation.
+
+Instead of opening one of these links, Alice prefers to consult the
+metadata provided by the policy:
+
+```shell
+$ kwctl inspect registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.4
+``` 
+
+This command produces the following output:
+
+![Output of kwctl inspect](/images/2021-june-kwctl-intro-admin-inspect-output.png)
+
+> **Note well:** a Kubewarden policy is a simple "`.wasm`" file that contains
+> both the policy's bytecode and its additional metadata.
+>
+> This allows to have everything in a single place, versioned together and
+> available off-line.
+
+## Evaluating the policy
+
+Now Alice wants to assess the reliability of the policy, plus find the right
+configuration values to satisfy her requirements.
+
+There are some good news for Alice: she doesn't need to deploy the policy into
+a Kubernetes cluster to perform this kind of rapid iterations. Once again, kwctl
+can help her wit the `run` sub-command.
+
+The `kwctl run` command takes the following parameters:
+  * `-r <request-path>`: path to a file containing the Kubernetes Admission Request
+    to be evaluated
+  * `--settings-json "<JSON document>"`: a string containing the JSON
+    representation of policy's settings. It's also possible to use the
+    `-s <settings-path` flag to read the policy settings from a local file
+  * `<URI of the policy>`: the URI pointing to the policy to be used
+
+> **Note well:** unfortunately, obtaining Kubernetes Admission Requests
+> requires some extra work. This is a "Kubernetes problem", we plan to
+> address it in the near future.
+>
+> In the meantime, we will assume Alice has access to files like
+> [these ones](https://github.com/kubewarden/psp-capabilities/tree/main/test_data).
```

Comment:
Given that the tree can change over time, rendering this links unusable in the future is better to use GitHub "permalinks".

```suggestion
> [these ones](https://github.com/kubewarden/psp-capabilities/tree/a786aba746e807aa3c6121438e8c05e724400861/test_data).
```